### PR TITLE
Review screen

### PR DESCRIPTION
**Pull request by balogun14**

### CODE REVIEW

File: lib/generated/assets.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
+///This file is automatically generated. DO NOT EDIT, all your changes would be lost.
```
Comment (dro-1) [inline lib/generated/assets.dart:1]: What is the purpose of this file?

Comment (dro-1) [inline lib/generated/assets.dart:1]: What is the purpose of this file?

File: pubspec.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ publish_to: "none" # Remove this line if you wish to publish to pub.dev
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.17.3 <3.0.0"
+  sdk: ">=2.17.3 >3.0.0"
```
Comment (dro-1) [inline pubspec.yaml]: Why did you change this? Had to change it back to run the code on my laptop.

File: android/app/build.gradle [hunk 3]
```diff
@@ -27,7 +27,7 @@ apply from: "$flutterRoot/packages/flutter_tools/gradle/flutter.gradle"
 apply plugin: 'com.google.gms.google-services'
 
 android {
-    compileSdkVersion 33//flutter.compileSdkVersion
+    compileSdkVersion 34//flutter.compileSdkVersion
```
Comment (dro-1) [inline android/app/build.gradle:30]: Why did you change this value as well?

File: lib/apps/customer_app/features/review/ui/review.screen.dart [hunk 4]
```diff
@@ -0,0 +1,155 @@
+import 'package:flutter/cupertino.dart';
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/review.screen.dart:1]: This `ReviewScreen` was not added to the app router so I can't open it. Read this two files to figure out how to add it:
-  https://github.com/Handees/mobile/blob/dev/lib/shared/routes/routers.dart
-  https://github.com/Handees/mobile/blob/dev/lib/shared/routes/routes.dart

**Review by dro-1 — CHANGES_REQUESTED** [review]

File: lib/shared/routes/routers.dart [hunk 5]
```diff
@@ -148,6 +149,9 @@ final _customerAppRouter = NavRouter(
           fullscreenDialog: true,
         );
         break;
+      case CustomerAppRoutes.review:
```
Comment (dro-1) [inline lib/shared/routes/routers.dart:152]: You were supposed to implement this the way other pages like CustomerAppRoutes.artisanSwitch and CustomerAppRoutes.editPrimary were done and not like CustomerAppRoutes.pickService.

File: lib/generated/assets.dart (hunk 1 above)
Comment (dro-1) [reply lib/generated/assets.dart:1]: I'm still wondering how you came about this file?

File: lib/apps/customer_app/features/review/ui/review.screen.dart [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+class ReviewScreen extends StatefulWidget {
+  const ReviewScreen({super.key});
+
+  @override
+  _ReviewScreenState createState() => _ReviewScreenState();
+}
+
+class _ReviewScreenState extends State<ReviewScreen> {
+  int _rating = 0;
+  final Map<String, bool> _feedback = {
```
Comment (dro-1) [inline lib/apps/customer_app/features/review/ui/review.screen.dart:20]: You forgot to add these into the UI and the font of Service Fee and the likes don't look correct.

**Comment by balogun14** [comment]
@dro-1 
<a href="https://ibb.co/mSLYFKx">Screenshot-from-2024-04-19-16-22-06</a>

**Pull request merged**